### ISSUE TITLE
is it possible to use F1 score for only positive label for binary data ?

### ISSUE THREAD

**Issue by Sandy4321** [manuel-calzolari/sklearn-genetic#8]
is it possible to use F1 score for only positive label?
like
f1_score( y_test, current_prediction, pos_label = 'NO', average='binary')

**Comment by manuel-calzolari** [comment]
Sorry for the extremely late reply.

You can try something like this:
```
from sklearn.metrics import f1_score, make_scorer

...

selector = GeneticSelectionCV(estimator, scoring=make_scorer(f1_score, pos_label='NO', average='binary'))
```

Please feel free to reopen the issue if necessary.
